package.json

```diff
@@ -65,13 +65,14 @@
   },
   "scripts": {
     "build": "node scripts/build.js",
-    "solidity:compile": "node solidity/compile.js",
+    "solidity:clean": "node solidity/scripts/clean.js",
```

Comment:
I like how you moved the solidity scripts to a separate directory. It's a step in the right direction for making the solidity section an independent module.